src/mito2/src/engine/tests.rs

```diff
@@ -285,3 +285,63 @@ async fn test_write_to_region() {
     };
     assert_eq!(num_rows, rows_inserted);
 }
+
+#[tokio::test]
+async fn test_region_replay() {
+    common_telemetry::init_default_ut_logging();
+    let env = TestEnv::with_prefix("region-replay");
+    let (engine, logstore, object_store) = env.create_engine(MitoConfig::default()).await;
```

Comment:
Could we keep the logstore and object store in the `TestEnv`? Then we can get them via getters and don't need to return them in `create_engine`